### PR TITLE
CWI-715: updated search for default templates

### PR DESCRIPTION
**Pull request by KatieHendricksen**
https://mural.atlassian.net/browse/CWI-715

We updated the search to include searching for default templates.

### CODE REVIEW

File: packages/mural-picker/src/components/mural-picker/index.tsx [hunk 1]
```diff
@@ -262,12 +262,20 @@ const MuralPicker = ({
...
   }, [rooms]);
 
-  const templateSearch = (q?: string) => {
-    if (q) {
-      const qEncoded = encodeURIComponent(q);
-      return apiClientRef.current.searchCrossWorkspaceTemplates({
-        q: qEncoded,
-      });
+  const templateSearch = async (searchKeyword?: string) => {
+    let templateResults: Template[] | TemplateSummary[];
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-picker/index.tsx:266]: **Nitpick**: this variable can be removed

File: packages/mural-picker/src/components/mural-picker/index.tsx [hunk 2]
```diff
@@ -262,12 +262,20 @@ const MuralPicker = ({
...
+    if (searchKeyword) {
+      // eslint-disable-next-line no-shadow
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-picker/index.tsx:268]: Why do we need this eslint exception?

File: packages/mural-picker/src/components/mural-picker/index.tsx (hunk 1 above)
Comment (KatieHendricksen) [reply packages/mural-picker/src/components/mural-picker/index.tsx]: good idea!

File: packages/mural-picker/src/components/mural-picker/index.tsx (hunk 2 above)
Comment (KatieHendricksen) [reply packages/mural-picker/src/components/mural-picker/index.tsx:267]: default templates is declared in the upper scope, I was going to check if we should change one of the names or if this is ok?

Comment (KavenTheriault) [reply packages/mural-picker/src/components/mural-picker/index.tsx:267]: Yes, just rename it here

File: runtimes/16.13.1/16.13.1 [hunk 3]
```diff
@@ -0,0 +1 @@
+runtimes/16.13.1
```
Comment (carlo-808) [inline runtimes/16.13.1/16.13.1:1]: I'm not sure if we need to commit this change.

**Review by carlo-808 — APPROVED** [review]
Looks good! 👍

**Pull request merged**